src/library_signals.js

```diff
@@ -37,6 +37,16 @@ var LibrarySignals = {
 #if ASSERTIONS
     err('Calling stub instead of kill()');
 #endif
+    if (sig < 0 || sig >= 65/*_NSIG*/) return {{{ cDefine('EINVAL') }}};
+    setErrNo({{{ cDefine('EPERM') }}});
+    return -1;
+  },
+
+  pthread_kill: function(thread, sig) {
+#if ASSERTIONS
+    err('Calling stub instead of pthread_kill()');
+#endif
+    if (sig < 0 || sig >= 65/*_NSIG*/) return {{{ cDefine('EINVAL') }}};
```

Comment:
Do you think we should still check for invalid threads and for `0` signals? For example:
```suggestion
    if (sig < 0 || sig >= 65/*_NSIG*/) return {{{ cDefine('EINVAL') }}};
    if (!thread || {{{ makeGetValue('thread', C_STRUCTS.pthread.self, 'i32') }}} !== thread) return {{{ cDefine('ESRCH') }}};
    if (sig === 0) return 0; // signal == 0 is a no-op.
```
(this seems to fix the `posixtest.test_pthread_kill*` test failures)